build/run_build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+echo "Office Answers Project"
+echo "----------------------"
+
+py=python3
+py36_error_msg="Python 3.6 is not installed.\nPlease install or modify this script to use proper Python executable"
+
+if command -v $py &>/dev/null; then
```

Comment:
you could pass this step by just using conda for managing the virtualenv in the future

but no need to change it :)